cmd/describe.go

```diff
@@ -0,0 +1,55 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/redhat-developer/odo/pkg/application"
+	"github.com/redhat-developer/odo/pkg/component"
+	"github.com/redhat-developer/odo/pkg/project"
+	"github.com/spf13/cobra"
+)
+
+var describeCmd = &cobra.Command{
+	Use:   "describe [component_name]",
+	Short: "Describe the given component",
+	Long:  `Describe the given component.`,
+	Example: `  # Describe nodejs component,
+    ocdev describe nodejs
+	`,
+	Run: func(cmd *cobra.Command, args []string) {
+		client := getOcClient()
+		// Application
+		currentApplication, err := application.GetCurrent(client)
+		checkError(err, "")
+		// Project
+		currentProject := project.GetCurrent(client)
+		var cmpnt string
+		if len(args) == 0 {
+			var err error
+			cmpnt, err = component.GetCurrent(client, currentApplication, currentProject)
+			checkError(err, "")
+		} else {
+			cmpnt = args[0]
+		}
+		path, componentURL, appStore, err := component.GetComponentDesc(client, cmpnt, currentApplication, currentProject)
+		checkError(err, "")
+		// Source
+		if path != "" {
+			fmt.Println("Component", cmpnt, "with source in", path)
+		}
+		// URL
+		if componentURL != "" {
+			fmt.Println("This Component is externally exposed via", componentURL)
+		}
+		// Storage
+		for _, store := range appStore {
+			fmt.Println("This Component uses storage", store.Name, "of size", store.Size)
```

Comment:
it would be useful to also display storage.Path

Reply:
@kadel , storage.List, don't give us path :(

Reply:
it's fixed in the unmount PR #349

Reply:
Ok, we can update in another PR when #349 is merged